cpp/arcticdb/util/test/gtest_main.cpp

```diff
@@ -7,7 +7,7 @@
 
 #include <gtest/gtest.h>
 #include <arcticdb/util/global_lifetimes.hpp>
-#include <pybind11/pybind11.h>  // Must not directly include Python.h on Windows
+#include <pybind11/pybind11.h>
```

Comment:
Nit: A potentially useful comment got deleted. Fine to leave as is if comment is not useful.